src/Components/CreateNewPoi/PoiThirdPageFields.js

```diff
@@ -25,7 +25,7 @@ const PoiThirdPageFields = ({
       {scoreRating.map(rating => (
```

Comment:
Could be one component for the three rates

Reply:
correction done